data/mods/blazemod/vehicleparts/blob_boards.json

```diff
@@ -0,0 +1,343 @@
+[
+  {
+    "id": "frostie_wall",
+    "type": "vehicle_part",
+    "name": "icy windshield",
+    "item": "frostie",
+    "location": "center",
+    "symbol": "\"",
+    "broken_symbol": "0",
+    "color": "light_blue",
+    "broken_color": "light_blue",
+    "durability": 500,
+    "damage_modifier": 80,
+    "folded_volume": 1,
+    "breaks_into": [ { "item": "frostiegrow" } ],
+    "description": "A living blob, existing in a super-cooled state.  It is transparent, and can be used to see out of the vehicle.",
+    "requirements": { "repair": { "time": "5 m", "using": [ [ "blobrep", 1 ] ] } },
+    "flags": [ "OBSTACLE", "WINDOW", "FOLDABLE" ]
+  },
+  {
+    "id": "gloopleboard_horizontal",
+    "type": "vehicle_part",
+    "name": "gel panel",
+    "item": "gloople",
+    "location": "center",
+    "symbol": "h",
+    "broken_symbol": "#",
+    "color": "light_green",
+    "broken_color": "light_green",
+    "durability": 250,
+    "damage_modifier": 10,
+    "folded_volume": 1,
+    "breaks_into": [ { "item": "glooplegrow" } ],
+    "requirements": { "repair": { "time": "5 m", "using": [ [ "blobrep", 1 ] ] } },
+    "flags": [ "OPAQUE", "OBSTACLE", "FOLDABLE", "FULL_BOARD" ]
+  },
+  {
+    "id": "gloopleboard_ne",
+    "type": "vehicle_part",
+    "copy-from": "gloopleboard_horizontal",
+    "symbol": "u"
+  },
+  {
+    "id": "gloopleboard_nw",
+    "type": "vehicle_part",
+    "copy-from": "gloople_horizontal",
+    "symbol": "y"
+  },
+  {
+    "id": "gloopleboard_se",
+    "type": "vehicle_part",
+    "copy-from": "gloople_horizontal",
```

Comment:
```suggestion
    "copy-from": "gloopleboard_horizontal",
```

Reply:
Oh snap I see what you mean. Changing em now, thanks for catching this.